<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="files">
       <attribute name="title">
        <string>Files</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLineEdit" name="textPathAdd">
          <property name="text">
           <string>Select an image sequence</string>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="addPath">
          <property name="maximumSize">
           <size>
            <width>151</width>
            <height>34</height>
           </size>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="openPath">
          <property name="maximumSize">
           <size>
            <width>151</width>
            <height>34</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>506</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="textBackgroundAdd">
          <property name="text">
           <string>Select a background image (optional)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="removePath">
          <property name="maximumSize">
           <size>
            <width>151</width>
            <height>34</height>
           </size>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" rowspan="3">
         <widget class="QTableWidget" name="tablePath">
          <column>
           <property name="text">
            <string>Movie to track</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Background</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QPushButton" name="startButton">
          <property name="text">
           <string>Start tracking</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="Line" name="line_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="openBackground">
          <property name="maximumSize">
           <size>
            <width>151</width>
            <height>34</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="displayPanel">
       <attribute name="title">
        <string>Display</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="display">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="display2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="parameters">
       <attribute name="title">
        <string>Parameters</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QPushButton" name="setupWindow">
          <property name="text">
           <string>Setup help</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableParameters">
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>Parameter</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Description</string>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="replay">
       <attribute name="title">
        <string>Replay</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="2" column="5">
         <widget class="QPushButton" name="replayHelp">
          <property name="text">
           <string/>
          </property>
          <property name="shortcut">
           <string>H</string>
          </property>
         </widget>
        </item>
        <item row="16" column="11">
         <widget class="QCheckBox" name="replayNumbers">
          <property name="text">
           <string>Numbers</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="25" column="1" colspan="7">
         <widget class="QSlider" name="replaySlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="22" column="11">
         <widget class="QSpinBox" name="replayFps">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>25</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="19" column="11">
         <widget class="QSpinBox" name="replaySize">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>150</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="10" column="11">
         <widget class="QCheckBox" name="replayEllipses">
          <property name="text">
           <string>Ellipses</string>
          </property>
         </widget>
        </item>
        <item row="12" column="11">
         <widget class="Line" name="line_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="18" column="11">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Size:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QPushButton" name="replayPrevious">
           <property name="maximumSize">
            <size>
             <width>78</width>
             <height>28</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="shortcut">
            <string>S</string>
           </property>
          </widget>
          <widget class="QPushButton" name="replayNext">
           <property name="maximumSize">
            <size>
             <width>78</width>
             <height>28</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="shortcut">
            <string>Z</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="8" column="11">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Options</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="20" column="11">
         <widget class="Line" name="line_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="11" column="11">
         <widget class="QComboBox" name="ellipseBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="25" column="10" colspan="2">
         <widget class="QPushButton" name="playReplay">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>79</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>79</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Space</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoRepeatDelay">
           <number>40</number>
          </property>
          <property name="autoRepeatInterval">
           <number>40</number>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="21" column="11">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Frame rate:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="8">
         <widget class="QLineEdit" name="replayPath">
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="25" column="8" colspan="2">
         <widget class="QLabel" name="replayNumber">
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
        <item row="15" column="11">
         <widget class="Line" name="line_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="24" column="1" colspan="11">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="11">
         <widget class="QPushButton" name="replayRefresh">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="9" column="11">
         <widget class="Line" name="line_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="11">
         <widget class="QPushButton" name="replayOpen">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="shortcut">
           <string>O</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="swapReplay">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>Swap</string>
          </property>
          <property name="shortcut">
           <string>Return</string>
          </property>
         </widget>
        </item>
        <item row="14" column="11">
         <widget class="QComboBox" name="arrowBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="13" column="11">
         <widget class="QCheckBox" name="replayArrows">
          <property name="text">
           <string>Arrows</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="11">
         <widget class="QPushButton" name="replaySave">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QComboBox" name="object1Replay">
           <property name="minimumSize">
            <size>
             <width>79</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>79</width>
             <height>26</height>
            </size>
           </property>
          </widget>
          <widget class="QComboBox" name="object2Replay">
           <property name="minimumSize">
            <size>
             <width>79</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>79</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </widget>
        </item>
        <item row="17" column="11">
         <widget class="QCheckBox" name="replayTrace">
          <property name="text">
           <string>Traces</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" rowspan="20" colspan="7">
         <widget class="QLabel" name="replayDisplay">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="6">
         <widget class="Line" name="line_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="6">
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="8" rowspan="23" colspan="2">
         <widget class="Line" name="line">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="11">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
